src/pages/[l1_l2]/[topic]/[...slug].tsx

```diff
@@ -254,21 +257,23 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 
     const tertiaryNavItems = await getSideNav(topicSlug);
 
-    const topicName = capitalizeFirstLetter(
-        pathComponents[pathComponents.length - 2]
-    );
+    const topicName = pathComponents[pathComponents.length - 2];
+
+    const metaInfoForTopic = await getMetaInfoForTopic(topicName);
 
     const contentTypeAggregateSlug = pillCategoryToSlug.get(contentType);
 
     const data = {
         contentType: contentType,
         tertiaryNavItems: tertiaryNavItems,
-        topicName: topicName,
+        topicName: capitalizeFirstLetter(topicName),
```

Comment:
Won't this lead to `full-text-search` appearing as `Full-text-search`. Should we be using `metaInfoForTopic.name`?